torch_frame/__init__.py

```diff
@@ -11,7 +11,7 @@
     embedding,
 )
 from .data import TensorFrame
-from .typing import TaskType, Metric, DataFrame, NAStrategy
+from .typing import TaskType, Metric, DataFrame, NAStrategy, TensorData
```

Comment:
Where is this helpful?

Reply:
In `eq` function used by tests, but we can move that into package.